From 778bf1125febabe180687a5df41acf76222097d6 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Tue, 28 Aug 2012 22:35:34 +0300
Subject: [PATCH 335/370] conf: put mnemonics near the magic values

RB: r/981/diff/3/?file=27987#file27987line43

+ Shorten the names of private magic values.
---
 conf/obj_ops.c | 38 ++++++++++++++++++--------------------
 conf/obj_ops.h |  2 +-
 2 files changed, 19 insertions(+), 21 deletions(-)

diff --git a/conf/obj_ops.c b/conf/obj_ops.c
index 25bdb6c..228bcdc 100644
--- a/conf/obj_ops.c
+++ b/conf/obj_ops.c
@@ -46,28 +46,26 @@ static bool generic_obj_invariant(const void *bob);
 static bool buf_is_valid(const struct c2_buf *buf);
 
 enum {
-	CONF_GENERIC_MAGIC    = 0x33ca11ab1ed0d077,
-	CONF_DIR_MAGIC        = 0x3301dca1ab005e77,
-	CONF_PROFILE_MAGIC    = 0x33c105ab1e5ea177,
-	CONF_FILESYSTEM_MAGIC = 0x33f00d0fc01aba77,
-	CONF_SERVICE_MAGIC    = 0x33b1a5ed10ca1e77,
-	CONF_NODE_MAGIC       = 0x33c01055a1d05a77,
-	CONF_NIC_MAGIC        = 0x33ba5eba11feed77,
-	CONF_SDEV_MAGIC       = 0x33a115eed5a1ad77,
-	CONF_PARTITION_MAGIC  = 0x33bacca15a100f77
+	DIR_MAGIC        = 0x3301dca1ab005e77, /* old calaboose */
+	PROFILE_MAGIC    = 0x33c105ab1e5ea177, /* closable seal */
+	FILESYSTEM_MAGIC = 0x33f00d0fc01aba77, /* food of Colaba */
+	SERVICE_MAGIC    = 0x33b1a5ed10ca1e77, /* biased locale */
+	NODE_MAGIC       = 0x33c01055a1d05a77, /* colossal dosa */
+	NIC_MAGIC        = 0x33ba5eba11feed77, /* baseball feed */
+	SDEV_MAGIC       = 0x33a115eed5a1ad77, /* allseed salad */
+	PARTITION_MAGIC  = 0x33bacca15a100f77  /* bacca is aloof */
 };
 
 static const struct c2_bob_type generic_obj_bob = {
 	.bt_name         = "c2_conf_obj",
 	.bt_magix_offset = C2_MAGIX_OFFSET(struct c2_conf_obj, co_gen_magic),
-	.bt_magix        = CONF_GENERIC_MAGIC,
+	.bt_magix        = C2_CONF_OBJ_MAGIC,
 	.bt_check        = generic_obj_invariant
 };
 C2_BOB_DEFINE(static, &generic_obj_bob, c2_conf_obj);
 
 C2_TL_DESCR_DEFINE(generic_objs, "c2_conf_obj-s", static, struct c2_conf_obj,
-		   co_linkage, co_gen_magic, CONF_GENERIC_MAGIC,
-		   CONF_DIR_MAGIC);
+		   co_linkage, co_gen_magic, C2_CONF_OBJ_MAGIC, DIR_MAGIC);
 C2_TL_DEFINE(generic_objs, static, struct c2_conf_obj);
 
 #define _BOB_TYPE_DEFINE(abbrev, magic)                            \
@@ -91,14 +89,14 @@ const struct c2_bob_type c2_conf_ ## abbrev ## _bob = {            \
 	.bt_check = abbrev ## _invariant                           \
 }
 
-_BOB_TYPE_DEFINE(dir,        CONF_DIR_MAGIC);
-_BOB_TYPE_DEFINE(profile,    CONF_PROFILE_MAGIC);
-_BOB_TYPE_DEFINE(filesystem, CONF_FILESYSTEM_MAGIC);
-_BOB_TYPE_DEFINE(service,    CONF_SERVICE_MAGIC);
-_BOB_TYPE_DEFINE(node,       CONF_NODE_MAGIC);
-_BOB_TYPE_DEFINE(nic,        CONF_NIC_MAGIC);
-_BOB_TYPE_DEFINE(sdev,       CONF_SDEV_MAGIC);
-_BOB_TYPE_DEFINE(partition,  CONF_PARTITION_MAGIC);
+_BOB_TYPE_DEFINE(dir,        DIR_MAGIC);
+_BOB_TYPE_DEFINE(profile,    PROFILE_MAGIC);
+_BOB_TYPE_DEFINE(filesystem, FILESYSTEM_MAGIC);
+_BOB_TYPE_DEFINE(service,    SERVICE_MAGIC);
+_BOB_TYPE_DEFINE(node,       NODE_MAGIC);
+_BOB_TYPE_DEFINE(nic,        NIC_MAGIC);
+_BOB_TYPE_DEFINE(sdev,       SDEV_MAGIC);
+_BOB_TYPE_DEFINE(partition,  PARTITION_MAGIC);
 #undef _BOB_TYPE_DEFINE
 
 /** Configuration object type. */
diff --git a/conf/obj_ops.h b/conf/obj_ops.h
index 5981362..2af46a4 100644
--- a/conf/obj_ops.h
+++ b/conf/obj_ops.h
@@ -67,7 +67,7 @@ C2_TL_DECLARE(c2_conf_reg, , struct c2_conf_obj);
 struct c2_conf_reg;
 
 /** Magic value for a generic configuration object (c2_conf_obj). */
-enum { C2_CONF_OBJ_MAGIC = 0x335e1f1e55ce1177 };
+enum { C2_CONF_OBJ_MAGIC = 0x335e1f1e55ce1177 /* selfless cell */ };
 
 /** Configuration object operations. */
 struct c2_conf_obj_ops {
-- 
1.8.3.2

